app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -64,13 +64,7 @@ protected function validator(array $data)
     protected function create(array $data)
     {
         $user = User::get();
-        // dd(count($user));
-        if (count($user) == 0) {
-            $admin = True;
-        } else {
-            $admin = False;
-        }
-        // dd($admin);
+        count($user) == 0 ? $admin = True : $admin = False;
```

Comment:
or better way is `$admin = count($user) == 0 ? true : false;`

Note: `true` or `false` not `True` or `False`